linguistics/src/test/java/com/yahoo/language/significance/DefaultSignificanceModelTest.java

```diff
@@ -0,0 +1,29 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.language.significance;
+
+import org.junit.Test;
```

Comment:
I prefer that we use Junit5 for new units tests. I'll help you add that to linguistics if not present already.